lms/djangoapps/course_api/blocks/tests/test_views.py

```diff
@@ -203,10 +204,14 @@ def test_block_counts_param(self):
             )
 
     def test_student_view_data_param(self):
-        response = self.verify_response(params={'student_view_data': ['video', 'chapter']})
+        response = self.verify_response(params={'student_view_data': ['video', 'chapter', 'discussion']})
```

Comment:
Change this to `params={'student_view_data': BLOCK_TYPES_WITH_STUDENT_VIEW_DATA + ['chapter']}`.